tests/vmi_multiqueue_test.go

```diff
@@ -20,22 +20,63 @@
 package tests_test
 
 import (
+	"encoding/json"
 	"encoding/xml"
 	"fmt"
+	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
 	k8sv1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 
 	v1 "kubevirt.io/client-go/api/v1"
 	"kubevirt.io/client-go/kubecli"
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 	"kubevirt.io/kubevirt/tests"
 )
 
+func setSpcT(virtClient *kubecli.KubevirtClient) *string {
+	kubeVirtConfig, err := (*virtClient).CoreV1().ConfigMaps(tests.KubeVirtInstallNamespace).Get("kubevirt-config", metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred())
+	originalContext, present := kubeVirtConfig.Data[virtconfig.SELinuxLauncherTypeKey]
+	kubeVirtConfig.Data[virtconfig.SELinuxLauncherTypeKey] = "spc_t"
+	newData, err := json.Marshal(kubeVirtConfig.Data)
+	Expect(err).ToNot(HaveOccurred())
+	data := fmt.Sprintf(`[{ "op": "replace", "path": "/data", "value": %s }]`, string(newData))
+	kubeVirtConfig, err = (*virtClient).CoreV1().ConfigMaps(tests.KubeVirtInstallNamespace).Patch("kubevirt-config", types.JSONPatchType, []byte(data))
+	Expect(err).ToNot(HaveOccurred())
+	// Allow time for virt-controller's ConfigMap cache to sync
+	time.Sleep(3 * time.Second)
```

Comment:
I think yon replace this with 
```suggestion
tests.UpdateClusterConfigValueAndWait(virtconfig.SELinuxLauncherTypeKey, string(newData)
```

Reply:
Thank you, I knew there had to be a simpler way!
Will fix.